internal/ignition/ignition.go

```diff
@@ -1107,15 +1072,6 @@ func setFileInIgnition(config *config_latest_types.Config, filePath string, file
 	config.Storage.Files = append(config.Storage.Files, file)
 }
 
-func setUnitInIgnition(config *config_latest_types.Config, contents, name string, enabled bool) {
```

Comment:
actually , this is a usefull function. We may not need it now, but maybe later we will need to add some systemd unit to ignition. What do you think about moving this function and setFileInIgnition to a different file and keep them there?

Reply:
Not sure I understand. I can't leave this function because linter complains that we have an unused function in the codebase. Any idea how to overcome it? I totally agree it may become useful in the future, but I'm not sure how to embed it into the current flow

Reply:
we can probably use something like:
//lint:ignore U1000 Ignore unused function 
but it is too much bother, lets remove it

Reply:
Good, that was fairly easy. I'll keep the function